feat: support non drs environments (#471)
vmware-samples/packer-examples-for-vsphere: README.md

vsphere_folder               = "sfo-w01-fd-templates"
```

> > **Note**

Reviewer: In the example above, you should add 

```hcl
vsphere_endpoint             = "sfo-w01-vc01.sfo.rainpole.io"
vsphere_username             = "svc-packer-vsphere@rainpole.io"
vsphere_password             = "<plaintext_password>"
vsphere_insecure_connection  = true
vsphere_datacenter           = "sfo-w01-dc01"
vsphere_cluster              = "sfo-w01-cl01"
//vsphere_host               = "sfo-w01-esx01"
vsphere_datastore            = "sfo-w01-cl01-ds-vsan01"
vsphere_network              = "sfo-w01-seg-dhcp"
vsphere_folder               = "sfo-w01-fd-templates"
 ```

>

> > When targeting standalone hosts or clusters without DRS you must set the `vsphere_host` variable.

Reviewer: ```suggestion
> When targeting standalone ESXi hosts or vSphere clusters with vSphere DRS disabled, you must set the `vsphere_host` variable.
```

>

> > **Example** (Clusters without DRS):

Reviewer: ```suggestion
> **Example** (vSphere Clusters with vSphere DRS Disabled):
```

>
> ```hcl
> ...
> vsphere_datacenter           = "sfo-w01-dc01"
> vsphere_cluster              = "sfo-w01-cl01"
> vsphere_host                 = "sfo-w01-esx01"
> vsphere_folder               = "sfo-w01-fd-templates"
> ...
> ```
>

> > **Example** (vCenter Managed Standalone Hosts):

Reviewer: ```suggestion
> **Example** (Standalone ESXi Hosts Managed by vCenter Server):
```

>

> > For vCenter managed standalone hosts, comment or remove `vsphere_cluster`.

Reviewer: ```suggestion
> For a standalone ESXI host managed by vCenter Server, comment or remove `vsphere_cluster`.
```

>
> ```hcl
> ...
> vsphere_datacenter           = "sfo-w01-dc01"

> > //vsphere_cluster              = "sfo-w01-cl01"

Reviewer: ```suggestion
> //vsphere_cluster            = "sfo-w01-cl01"
```

> vsphere_host                 = "sfo-w01-esx01"
> vsphere_folder               = "sfo-w01-fd-templates"
> ...
> ```
>

> > **Example** (Standalone Hosts):

Reviewer: ```suggestion
> **Example** (Standalone ESXi Host):
```

>

> > For standalone hosts, comment or remove `vsphere_datacenter`, `vsphere_cluster` and `vsphere_folder`.

Reviewer: ```suggestion
> For a standalone, unmanaged ESXi host, comment or remove `vsphere_datacenter`, `vsphere_cluster`, and `vsphere_folder`.
```

>
> ```hcl
> ...

> > //vsphere_datacenter           = "sfo-w01-dc01"

Reviewer: ```suggestion
> //vsphere_datacenter         = "sfo-w01-dc01"
```

> > //vsphere_cluster              = "sfo-w01-cl01"

Reviewer: ```suggestion
> //vsphere_cluster            = "sfo-w01-cl01"
```

> vsphere_host                 = "sfo-w01-esx01"

> > //vsphere_folder               = "sfo-w01-fd-templates"

Reviewer: ```suggestion
> //vsphere_folder             = "sfo-w01-fd-templates"
```
